src/main/java/mil/dds/anet/beans/Task.java

```diff
@@ -185,6 +189,28 @@ public Organization getResponsibleOrg() {
     return responsibleOrg.getForeignObject();
   }
 
+  @GraphQLQuery(name = "positions")
```

Comment:
`Position` is just the data type, the graphql name should reflect the relationship. In this case, I suggest `responsibles` or `responsiblePositions`